### PR TITLE
chore: readme and others

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -3,21 +3,24 @@ DB_HOST=http://clickhouse
...
 DB_NAME=stats
+# Logs
+# log format can be 'simple' or 'json'
+LOG_FORMAT=simple
 # select a slot as close as possible to the last finalized: find last finalized epoch, copy last slot from it
 START_EPOCH=155000
 # network_name will be mainnet or goerli (testnet)
 # you can set URL list (it must be a comma separated string)
 EL_RPC_URLS=https://<network_name>.infura.io/v3/<secret>
+# network_name will be mainnet or goerli (testnet)
```
Comment (madlabman) [inline .env.example:14]: Duplicates.

File: README.md [hunk 2]
```diff
@@ -23,15 +23,19 @@ and when the application completes its first cycle, you can restart your instanc
 
 1. Use `.env.example` file content to create your own `.env` file
 2. Build app image via `docker-compose build app`
-3. Create `.volumes` directory from `docker` directory:
+3. Set owner for validators registry sources
+```bash
+chown -R 1000:1000 ./docker/validators
```
Comment (madlabman) [inline README.md:28]: Didn't get the motivation.

File: .env.example (hunk 1 above)
Comment (vgorkavenko) [reply .env.example]: fixed

File: README.md (hunk 2 above)
Comment (madlabman) [reply README.md:28]: It seems that the user has a different id, nice.

**Review by madlabman — APPROVED** [review]

**Pull request merged**